index.html

```diff
@@ -3,9 +3,94 @@
   <head>
     <link rel="stylesheet" href="app.css"
     <meta charset="utf-8">
-    <title></title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.css">
+    <title>Surf & Paddle Co</title>
   </head>
   <body>
+<div class="container">
+  <ul class="left-top-logo">
+<li class="fa fa-pie-chart"></li>
+<li class="surf">SURF & PADDLE CO. BLOG</li>
+</ul>
+</nav>
+    <ul class="right-top-nav">
+     <li>ABOUT</li>
+      <li>SEARCH</li>
+      <li><i class="fa fa-search"></i></li>
+    </ul>
+  </nav>
+  <div class="header">
+    <h1 class="main-title">SURF & PADDLE CO.</h1>
+    <p class="title-sub">An Ironyard Academy Project Est. Fall 2013</p>
+  </div>
+</div>
+<section class="left-section">
+   <h1 class="tit1">A Fantastic Engaging Storyline Goes Here</h1>
+   <img class="icon-section" src="http://placehold.it/25/25" alt="Icon"/>
```

Comment:
All of the author info should probably be grouped together in a `p` tag or something similar. Then you can use spans to set the author name, and date apart. the date might even be a link (`a`) so then the default browser style makes it underlined and you won't have to do too much else.